src/app/shared/cookie-popup/cookie-popup.html

```diff
@@ -5,9 +5,8 @@
     <a
       href="https://policies.google.com/technologies/cookies"
       mat-button
-      color="accent"
       target="_blank"
       rel="noopener">More details</a>
-    <button mat-button color="accent" (click)="accept()">Ok, Got it</button>
+    <button mat-button (click)="accept()">Ok, Got it</button>
```

Comment:
Can we use `color="primary"` here and above and not need the extra CSS? or does that use a hue that is too light in dark mode?

Reply:
`color="primary"` in light mode have insufficient contrast

![Screenshot from 2021-06-07 22-43-34](https://user-images.githubusercontent.com/48172978/121068978-2c9daf80-c7e2-11eb-8da7-5fd743d13cb5.png)